### DISCUSSION TITLE
Cells with neighbors?

### DISCUSSION THREAD

**Discussion by i-rember** [TodePond/CellPond#244] [Ideas]
I suggest the addition of a hexagon shape, which you can put a square on like a paddle, and a rectangle on like a square's color picker. Any stamped square of the same color as the square on the hexagon will check if the number of neighbors of the same color is equal to the rectangle's value. If not, the rule will not apply. 

You can also put multiple rectangles on the hexagon, for multiple numbers (or groups of numbers) of neighbors to check.

Edit: it might be a different shape because hexagons were already implemented.

**Comment by TodePond** [comment]
Hey that's a cool idea! Something like that could help to make rules where it sums up neighbours! I'll make an issue for it: https://github.com/TodePond/CellPond/issues/245
